### ISSUE TITLE
Evaluation of the real-world ScanNet dataset

### ISSUE THREAD

**Issue by yifliu3** [CuiRuikai/Partial2Complete#3]
Hi,

I find that you also evaluate the framework on ScanNet dataset. As the dataset contains raw partial point clouds, I'm confused about how you reproduce the supervised methods in Table 3 of the paper. 

Besides, the chair instances in ScanNet contain a diverse number of points, so I think an appropriate number of input points number should be defined. Could you please share more details of the ScanNet settings?

**Comment by CuiRuikai** [comment]
The supervised as well as unpaired methods are pre-trained with 3D EPN dataset. We provided more dataset details in supplementary material. Basically, we define the number of input points as 2048. If it is lower than 2048 we pad it with 0; if higher, then downsample it.

For data processing, we used MMDetection3D to load and process data. We extract objects using the class label and further segment then by instance label. We will release the code soon.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Yifan LIU ***@***.***>
Sent: Monday, September 4, 2023 21:20
To: CuiRuikai/Partial2Complete ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [CuiRuikai/Partial2Complete] Evaluation of the real-world ScanNet dataset (Issue #3)


Hi,

I find that you also evaluate the framework on ScanNet dataset. As the dataset contains raw partial point clouds, I'm confused about how you reproduce the supervised methods in Table 3 of the paper.

Besides, the chair instances in ScanNet contain a diverse number of points, so I think an appropriate number of input points number should be defined. Could you please share more details of the ScanNet settings?

**Comment by yifliu3** [comment]
Thanks a lot! 

Another question is that I notice the number of output points the model uses is also 2048 for ScanNet objects. I think it's more reasonable to set the output numbers larger than the input, so why do you choose 2048 rather than other numbers?

**Comment by yifliu3** [comment]
Thanks a lot your prompt reply. My problems got solved.
